devcycle_python_sdk/models/user.py

```diff
@@ -101,31 +101,40 @@ def create_user_from_context(
     ) -> "DevCycleUser":
         """
         Builds a DevCycleUser instance from the evaluation context. Will raise a TargetingKeyMissingError if
-        the context does not contain a valid targeting key or user_id attribute
+        the context does not contain a valid targeting key, user_id, or userId attribute
 
         :param context: The evaluation context to build the user from
         :return: A DevCycleUser instance
         """
         user_id = None
+        user_id_source = None
 
         if context:
             if context.targeting_key:
                 user_id = context.targeting_key
+                user_id_source = "targeting_key"
             elif context.attributes and "user_id" in context.attributes.keys():
                 user_id = context.attributes["user_id"]
+                user_id_source = "user_id"
+            elif context.attributes and "userId" in context.attributes.keys():
+                user_id = context.attributes["userId"]
+                user_id_source = "userId"
 
         if not user_id or not isinstance(user_id, str):
             raise TargetingKeyMissingError(
-                "DevCycle: Evaluation context does not contain a valid targeting key or user_id attribute"
+                "DevCycle: Evaluation context does not contain a valid targeting key, user_id, or userId attribute"
             )
 
         user = DevCycleUser(user_id=user_id)
         custom_data: Dict[str, Any] = {}
         private_custom_data: Dict[str, Any] = {}
         if context and context.attributes:
             for key, value in context.attributes.items():
+                # Skip user_id and userId - these are reserved for user ID mapping
                 if key == "user_id":
                     continue
+                if key == "userId":
+                    continue
```

Comment:
This doesn't handle targeting_key?

Reply:
it can be tho - we handle for this in the other languages afaik

Reply:
yeah agree we should add it here